app/domain/operations/ffe/migrate_application.rb

```diff
@@ -0,0 +1,744 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+require 'aca_entities'
+require 'aca_entities/ffe/operations/process_mcr_application'
+require 'aca_entities/ffe/transformers/mcr_to/family'
+require 'aca_entities/atp/transformers/cv/family'
+require 'aca_entities/atp/operations/family'
+require 'aca_entities/serializers/xml/medicaid/atp'
+
+# rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/ClassLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
why did we disable these rubocop checks?

Reply:
This operation is currently used for only importing MCR/ATP data, this operation needs to be scaled down to sub-operations, which we are gone do in follow up PR.

Reply:
okay so to confirm, this file is very, very large and we've disabled rubocop checks to prevent the error, but we're going to refactor it into smaller files in the future?

Reply:
Yes! Mark